websites/A/Amazon Kindle/presence.ts

```diff
@@ -0,0 +1,31 @@
+const presence = new Presence({
+		clientId: "1208798725116395592",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Stöbert in der Kindle Bibliothek",
```

Comment:
Don't think this should be in german...

Reply:
You are correct.
This was in when i originally wanted to make it for the german version online